src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        int supplySum = 0;
```

Comment:
https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#divide-program-logic

Please split logic to 3 separate private methods (reading file content, forming report and writing report to file) and reuse these method in public one